client/src/components/overview/details/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+function Details({ product }) {
+  const featureCreator = (feat) => `${feat.feature}   --   ${feat.value}`;
+
+  return (
+    <div id="ProductDetails">
+      <div className="productDetails">
+        <h2>
+          {product.slogan}
+        </h2>
+        <p>
+          {product.description}
+        </p>
+      </div>
+      <div className="features">
+        {product.features.map((feat, idx) => (
+          <p className="feature" key={idx}>
```

Comment:
I think you can use <ul> and <li> tags instead of <p> tags, they might be semantically more suitable for displaying a list of features.